### PR TITLE
IDVA5-126-sign-out

### PR DESCRIPTION
**Pull request by abhiahirv1**
sign out page

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by sardhani-ch — APPROVED** [review]

File: src/main/common/__utils/constants.ts [hunk 1]
```diff
@@ -17,3 +17,4 @@ export const BUSINESS_ADDRESS = "businessAddress";
 export const ANSWER_DATA = "answerData";
 export const GET_ACSP_REGISTRATION_DETAILS_ERROR: string = "Unable to get registration details for the transaction";
 export const POST_ACSP_REGISTRATION_DETAILS_ERROR: string = "Unable to post registration details for the transaction";
+export const PREVIOUSPAGEURL: string = "previouspageurl";
```
Comment (ttingle-ch) [inline src/main/common/__utils/constants.ts:20]: Other variables are have words separated by underscores could this made to be the same?

File: src/test/src/controllers/signOutController.test.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import { SIGN_OUT_URL, BASE_URL } from "../../../main/types/pageURL";
+
+jest.mock("@companieshouse/api-sdk-node");
+const router = supertest(app);
+
+describe("GET" + SIGN_OUT_URL, () => {
+    it("should return status 200", async () => {
+        await router.get(BASE_URL + SIGN_OUT_URL).expect(200);
+        expect(mocks.mockSessionMiddleware).toHaveBeenCalled();
+        expect(mocks.mockAuthenticationMiddleware).toHaveBeenCalled();
```
Comment (ttingle-ch) [inline src/test/src/controllers/signOutController.test.ts:13]: Can we add an expect for the page heading?

File: src/test/src/controllers/signOutController.test.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    });
+});
+
+// Test will return 302 after redirecting to the next page.
+describe("POST " + SIGN_OUT_URL, () => {
+    it("should return status 302 after redirect", async () => {
+        // Make a POST request with signout: "yes" to trigger redirection
+        await router.post(BASE_URL + SIGN_OUT_URL)
+            .send({ signout: "yes" })
+            .expect(302);
```
Comment (ttingle-ch) [inline src/test/src/controllers/signOutController.test.ts:23]: Can we add an expect for the url of the page?

File: src/main/common/__utils/constants.ts (hunk 1 above)
Comment (abhiahirv1) [reply src/main/common/__utils/constants.ts]: Done

File: src/test/src/controllers/signOutController.test.ts (hunk 2 above)
Comment (abhiahirv1) [reply src/test/src/controllers/signOutController.test.ts:14]: Done

File: src/test/src/controllers/signOutController.test.ts (hunk 3 above)
Comment (abhiahirv1) [reply src/test/src/controllers/signOutController.test.ts:24]: Done

**Review by ttingle-ch — APPROVED** [review]
LGTM

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Pull request merged**